pygmt/datasets/earth_relief.py

```diff
@@ -121,10 +143,23 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
             f"resolution '{resolution}' is not supported."
         )
 
+    with Session() as lib:
+        if Version(lib.info["version"]) < Version("6.4.0") and data_source != "igpp":
+            raise GMTVersionError(
+                f"The {data_source} option is not available for GMT"
+                " versions before 6.4.0."
+            )
```

Comment:
Because `gebco` is not the default option, I prefer to rewrite these lines to:
```suggestion
    if data_source == "gebco":
        with Session() as lib:
            if Version(lib.info["version"]) < Version("6.4.0"):
                raise GMTVersionError(
                    f"The {data_source} option is not available for GMT"
                    " versions before 6.4.0."
                )
```
In this way, we can avoid the version comparison for the default cases (i.e., `data_source="igpp"`).

Reply:
I went with this instead:

```
    earth_relief_sources = {"igpp": "earth_relief_", "gebco": "earth_gebco_"}
    if data_source in earth_relief_sources and data_source != "igpp":
        with Session() as lib:
            if Version(lib.info["version"]) < Version("6.4.0"):
                raise GMTVersionError(
                    f"The {data_source} option is not available for GMT"
                    " versions before 6.4.0."
                )
```

I'm not sure how other data sources will work with old versions of GMT, but I didn't want to have the only issue be raised if `gebco` is specified.